lib/test.js

```diff
@@ -19,7 +19,7 @@ try {
         it("should not create the park base successfully", function(done) {
             try {
                 parkBase = new Park();
-                assert(parkBase, "parkBase should not successfully construct");
```

Comment:
Note that this test used to not work at all-- if the base park *was* created successfully, the assert statement would have passed.